ci-operator/config/openshift/csi-operator/openshift-csi-operator-master.yaml

```diff
@@ -24,14 +44,37 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: unit
-  commands: make test
+- as: verify
+  commands: |
+    make verify
   container:
     from: src
-- as: verify
-  commands: make verify
+- as: unit
+  commands: |
+    make test-unit JUNITFILE=${ARTIFACT_DIR}/junit_report.xml
   container:
     from: src
+- as: e2e-aws-ovn-upgrade
+  steps:
+    cluster_profile: aws
+    workflow: openshift-upgrade-aws
+- as: e2e-aws-csi
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws-csi
```

Comment:
This one is failing because our [storage-conf-csi-aws-ebs](https://github.com/openshift/release/blob/master/ci-operator/step-registry/storage/conf/csi-aws-ebs/storage-conf-csi-aws-ebs-commands.sh) step expects the old directory in aws-ebs-csi-driver-operator-test image.

We need to update the test too (or the -test image?)

> /bin/bash: line 7: cd: /go/src/github.com/openshift/aws-ebs-csi-driver-operator: No such file or directory

Reply:
This is now fixed. See logs from https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/43919/rehearse-43919-pull-ci-openshift-csi-operator-master-e2e-aws-csi/1711527623693176832/artifacts/e2e-aws-csi/storage-conf-csi-aws-ebs/build-log.txt 

If it detects legacy directory, then the job will use that, otherwise it will fallback to older directory.

Reply:
That does not work on 4.14 and older, which have manifest still in `/go/src/github.com/openshift/aws-ebs-csi-driver-operator`